blazor/rich-text-editor/webassembly-performance.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: post
+title: WebAssembly Performance in Blazor RichTextEditor | Syncfusion
+description: This topic helps you to improve the performance of Web Assembly Application when using Syncfusion Blazor RichTextEditor components with some tips.
+platform: Blazor
+control: RichTextEditor
+documentation: ug
+---
+
+# WebAssembly Performance in Blazor RichTextEditor Component
+
+This section provides performance guidelines for using Syncfusion RichTextEditor component efficiently in Blazor WebAssembly application. The best practice or guidelines for general framework Blazor WebAssembly performance can be found [here](https://docs.microsoft.com/en-us/aspnet/core/blazor/webassembly-performance-best-practices).
+
+N> You can refer to our Getting Started with [Blazor Server-Side RichTextEditor](https://blazor.syncfusion.com/documentation/getting-started/blazor-server-side-visual-studio) and [Blazor WebAssembly RichTextEditor](./how-to/blazor-web-assembly) documentation pages for configuration specifications.
+
+## Avoid unnecessary component renders
+
+During Blazor Diffing Algorithm, every views of the RichTextEditor component and its child component will be checked for re-rendering. For instance, having **EventCallBack** on the application or RichTextEditor will check every child component, once event callback is completed.
+
+You can have fine-grained control over RichTextEditor component rendering. **PreventRender** method helps to avoid unnecessary re-rendering of the RichTextEditor component. This method internally overrides the **ShouldRender** method of the RichTextEditor to prevent rendering.
+
+In the following example:
+
+* **PreventRender** method is called in the **IncrementCount** method which is a click callback.
+* Now, RichTextEditor component will not be a part of the rendering which happens as result of the click event and **currentCount** alone will get updated.
```

Comment:
* Now, the RichTextEditor component will not be a part of the rendering, which happens as a result of the click event, and **currentCount** alone will get updated.

Reply:
addressed